src/app/app.module.ts

```diff
@@ -90,7 +91,8 @@ import { BeneficiariesEffects } from '@beneficiaries/beneficiaries.effects';
       useClass: ErrorInterceptor,
       multi: true,
     },
+    ExportCSVService,
```

Comment:
I think that is no need to add it here, because class already has `@Injectable` annotation.

Reply:
Can be removed